src/Components/Header/Banner.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Banner = ({ subHeading }) => {
```

Comment:
👍 for destructuring

Reply:
Lets talk a little more about arrow functions, which will help you improve the pure functions